Library/Formula/libgweather.rb

```diff
@@ -0,0 +1,99 @@
+class Libgweather < Formula
+  desc "GNOME library for weather, locations and timezones"
+  homepage "https://wiki.gnome.org/Projects/LibGWeather"
+  url "https://download.gnome.org/sources/libgweather/3.16/libgweather-3.16.1.tar.xz"
+  sha256 "db0955261427bb0f1a3f2747507863c591bf2f09dc477cfbf1cea0382cd34dfd"
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "gtk+3"
+  depends_on "geocode-glib"
+  depends_on "libsoup"
+  depends_on "gobject-introspection"
+  depends_on "vala" => :optional
+
+  def install
+    # ensures that the gobject-introspection files remain within the keg
+    inreplace "libgweather/Makefile.in", "@HAVE_INTROSPECTION_TRUE@girdir = $(INTROSPECTION_GIRDIR)",
```

Comment:
Just a nit but when `inreplace`-ing the same file, you can just do:

```ruby
    inreplace "libgweather/Makefile.in" do |s|
      s.gsub! "@HAVE_INTROSPECTION_TRUE@girdir = $(INTROSPECTION_GIRDIR)",
              "@HAVE_INTROSPECTION_TRUE@girdir = $(datadir)/gir-1.0"
      s.gsub! "@HAVE_INTROSPECTION_TRUE@typelibdir = $(INTROSPECTION_TYPELIBDIR)",
              "@HAVE_INTROSPECTION_TRUE@typelibdir = $(libdir)/girepository-1.0"
    end
```

I'll fix it on pull here.